src/utils/components/PendingChanges/utils/helpers.ts

```diff
@@ -150,11 +155,13 @@ export const getChangedNICs = (vm: V1VirtualMachine, vmi: V1VirtualMachineInstan
   const vmNICsNames = vmInterfaces?.map((nic) => nic?.name) || [];
   const vmiNICsNames = vmiInterfaces?.map((nic) => nic?.name) || [];
 
+  const autoAttach = getAutoAttachPodInterface(vm) !== false;
```

Comment:
I'm sorry. I missed this one. Could you please change this one to `autoAttachPodInterface`, too?